c++11/include/opentracing/stringref.h

```diff
@@ -1,6 +1,12 @@
 #ifndef OPENTRACING_STRINGREF_H
```

Comment:
Would you be open to something more like the C++17 string_view in terms of the API names?

http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2014/n3921.html